ui/perfherder/compare/CompareSubtestDistributionView.jsx

```diff
@@ -114,6 +116,8 @@ export default class CompareSubtestDistributionView extends React.Component {
     originalRevision,
     originalSubtestSignature,
   ) => {
+    const { failureMessages } = this.state;
```

Comment:
This should be used at line 153 in the present code

Reply:
I agree, it's not needed here.